www/include/configuration/configObject/contact/formContact.php

```diff
@@ -73,15 +73,15 @@
 /*
  * Check if this server is a Remote Server to hide some part of form
  */
-$DBRESULT = $pearDB->query("SELECT i.value FROM informations i WHERE i.key = 'isRemote'");
-$result = $DBRESULT->fetchRow();
+$dbResult = $pearDB->query("SELECT i.value FROM informations i WHERE i.key = 'isRemote'");
+$result = $dbResult->fetchRow();
```

Comment:
please replace fetchRow with fetch
```suggestion
$result = $dbResult->fetch();
```